src/components/OptimismDowntimeWarning/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Trans } from '@lingui/macro'
+import { SupportedChainId } from 'constants/chains'
+import { useActiveWeb3React } from 'hooks/web3'
+import { AlertOctagon } from 'react-feather'
+import styled from 'styled-components/macro'
+
+const Root = styled.div`
+  background-color: ${({ theme }) => theme.yellow3};
+  border-radius: 18px;
+  color: black;
+  margin-top: 16px;
+  max-width: 480px;
+  padding: 16px;
+  width: 100%;
+`
+const WarningIcon = styled(AlertOctagon)`
+  margin: 0 8px 0 0;
+`
+const TitleRow = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  margin: 0;
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 25px;
+`
+const Body = styled.div`
+  font-size: 12px;
+  line-height: 15px;
+  margin: 8px 0 0 0;
+`
+const LinkOutToNotion = styled.a`
+  color: black;
+`
+
+export default function OptimismDowntimeWarning() {
+  const { chainId } = useActiveWeb3React()
+  if (!chainId || ![SupportedChainId.OPTIMISM, SupportedChainId.OPTIMISTIC_KOVAN].includes(chainId)) {
+    return null
+  }
+  const name = 'Optimism'
+  return (
+    <Root>
+      <TitleRow>
+        <WarningIcon />
+        <Trans>{name} Scheduled Downtimes</Trans>
```

Comment:
you can just inline `{'Optimism'}` so it's clear that it's hard coded

Reply:
@justindomingue that works w/ your suggestion, right?

Reply:
It's neat. You should double check that it does extract the string properly with `yarn i18n:extract`. Love it if it works